index.css

```diff
@@ -0,0 +1,43 @@
+.main {
+  margin-top: 2rem;
+  display: flex;
+}
+
+.main section {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  flex-basis: 50%;
+  padding: 3rem;
+}
+
+.main section .brand-name {
+  color: var(--primary-color-dark);
+  font-size: 2.5rem;
+  font-weight: bold;
+  font-family: 'Quintessential', cursive;
+  letter-spacing: 0.5rem;
+}
+
+.main .main__intro-text {
+  font-weight: 500;
+  margin-top: 2rem;
+}
+
+.main .main__intro-subtext {
+  font-weight: 500;
+  margin-bottom: 2rem;
+}
+.main div {
+  flex-basis: 50%;
+}
+
+@media only screen and (max-width: 414px) {
```

Comment:
Can use variables instead of values in max-width. It would help in managing media queries if used in multiple pages.

Reply:
I think px is fine here. For contrast ratio, I need to make required changes in the component library colour css file.